htscodecs/rANS_static4x16pr.c

```diff
@@ -1199,11 +1201,20 @@ unsigned char *rans_compress_to_4x16(unsigned char *in, unsigned int in_size,
         int N = (order>>8) & 0xff;
         if (N == 0) N = 4; // default for compatibility with old tests
 
+        if (N > in_size)
+            N = in_size;
+
+        if (N > 255) {
```

Comment:
If this happens your CPU is broken, as `N` is masked above.  Note that this code isn't the same as the version in `arith_compress_to()`, where `N` currently could be greater than 255.

Reply:
And indeed the specification states it's one byte!  This is just an accident of duplication between the rans vs arith, where one is mistakenly not doing `& 0xff`.  Will fix.  Thanks